weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallSummary.tsx

```diff
@@ -34,6 +35,14 @@ export const CallSummary: React.FC<{
   );
   const costData = call.traceCall?.summary?.weave?.costs;
 
+  const inputBytes = call.traceCall?.inputs
```

Comment:
in case these things are huge, i would memoize this to avoid recalculating on each render

Reply:
good callout — thank you! updated :)